network/node/node.go

```diff
@@ -47,6 +47,9 @@ type Config struct {
 	ConfigFile string `json:"configFile"`
 	// May be nil.
 	CChainConfigFile string `json:"cChainConfigFile"`
+	// Flags can hold additional flags for the node.
+	// It can be empty.
+	Flags map[string]interface{} `json:"flags"`
```

Comment:
Can you document here that the precedence from highest to lowest is:
1, Config file
2. Flags defined in network.Config
3. Flags defined in node.Config